unlock-protocol-com/blog/email-airdrop-nft.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Unlock Protocol Email Airdropped NFTs
+subTitle: Unlock Protocol added a new feature to email a user when airdropping them an NFT from the creator dashboard.
+authorName: Angela Steffens
+publishDate: August 2, 2022
+description: Unlock Protocol added a new feature to email a user when airdropping them an NFT from the creator dashboard.
+image: /images/blog/email-airdrop-nft/airdrop-nft-share-img.png
+---
+
+The Unlock creator dashboard already included the functionality that allows someone logged in with the "Lock Manager" role to airdrop keys to a list of recipients. Now we've added the ability to send an email simultaneously!
+
+The email generated will also include a QR code of the signed NFT that they can use to prove that they own it when attending in-person events and a link to view their NFT. Let's take a look at the changes!
+
+## New field added to the airdrop screen
+A new field "email" is now present on the airdrop screen. 
+[airdrop screenshot with new email field](/img/blog/email-airdrop-nft/email-airdrop-screenshot1.png)
```

Comment:
clicking this URL throws a 404.

also, perhaps embed the image directly here, instead of linking to is?

Reply:
if this is intended to inline the image, needs to start with an !